packages/playground/src/components/contracts_list/contracts_table.vue

```diff
@@ -89,6 +89,27 @@
     </template>
   </weblet-layout>
 
+  <v-dialog width="800" v-model="contractStateDialog">
+    <v-card>
+      <v-card-title class="text-h5 mt-2"> Contract lock Details</v-card-title>
+      <v-card-text>
+        <v-row class="d-flex justify-center"> Amount Locked {{ contractLocked?.amountLocked }} TFTs. </v-row>
```

Comment:
Maybe it would look better if the amount was in a different color/style from the text next to it.

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/101194226/70e832d9-a07f-425e-bc07-3135f38689a5)

Not sure if it's the best option with balance but maybe it should be rounded to 4 decimals.

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/101194226/caa3c34a-8441-4318-ab26-dda540ab1d44)

Reply:
should be fixed